mono/mini/mixed_callstack_plugin.c

```diff
@@ -0,0 +1,191 @@
+#include "mixed_callstack_plugin.h"
+#include "mono/metadata/mono-debug.h"
+#include "mono/metadata/profiler.h"
+
+/* Dummy structure used for the profiler callbacks */
+typedef struct {
+	void* dummy;
+} PMIPProfiler;
+
+static char *
+pmip_pretty(MonoCompile* monoCompile)
+{
+	char* methodName;
+	char* assemblyName;
+	char* formattedPMIP;
+	MonoDebugSourceLocation* debugSourceLocation;
+	MonoDebugMethodInfo* debugMethodInfo;
+	MonoDomain* domain;
+	MonoMethod* method = monoCompile->method; 
+
+	domain = mono_domain_get();
+	if (!domain)
+		domain = mono_get_root_domain();
+
+	methodName = mono_method_full_name(method, TRUE);
+
+	debugSourceLocation = mono_debug_lookup_source_location(method, 0, domain);
+	debugMethodInfo = mono_debug_lookup_method(method);
+
+	assemblyName = method->klass->image->module_name;
+
+	formattedPMIP = g_strdup_printf("[%s] %s", assemblyName, methodName);
+
+	mono_debug_free_source_location(debugSourceLocation);
+	g_free(methodName);
+
+	return formattedPMIP;
+}
+
+#if !defined(DISABLE_JIT) && defined(HOST_WIN32)
+
+static gboolean enabled;
+static mono_mutex_t mutex;
+static HANDLE fileHandle;
+PMIPProfiler pmipProfiler;
+int pmipFileNum;
+
+#define pmip_my_callstack_lock() mono_os_mutex_lock (&mutex)
```

Comment:
Not a big deal, but why not change all naming if renaming?

Reply:
whoops.  missed one :/